docs/lexical/converters.mdx

```diff
@@ -6,14 +6,65 @@ desc: Conversion between lexical, markdown and html
 keywords: lexical, rich text, editor, headless cms, convert, html, mdx, markdown, md, conversion, export
 ---
 
+Lexical saves data in JSON - this is great for storage and flexibility and allows you to easily to convert it to other formats like JSX, HTML or Markdown.
+
+## Lexical => JSX
+
+If you have a React-based frontend, converting lexical to JSX is the recommended way to render rich text content in your frontend. To do that, import the `RichText` component from `@payloadcms/richtext-lexical/react` and pass the lexical content to it:
+
+```tsx
+import React from 'react'
+import { RichText } from '@payloadcms/richtext-lexical/react'
+
+export const MyComponent = ({ lexicalContent }) => {
+  return (
+    <RichText editorState={lexicalContent} />
+  )
+}
+```
+
+The `RichText` component will come with the most common serializers built-in, though you can also pass in your own serializers if you need to.
+
+### Converting Lexical Blocks to JSX
+
+In order to convert lexical blocks or inline blocks to JSX, you will have to add the `BlocksJSXConverter` to the RichText component. This converter is not included by default, as Payload doesn't know how to render your custom blocks.
+
+```tsx
+import React from 'react'
+import {
+  BlocksJSXConverter,
+  type JSXConvertersFunction,
+  RichText,
+} from '@payloadcms/richtext-lexical/react'
+
+const jsxConverters: JSXConvertersFunction = ({ defaultConverters }) => [
```

Comment:
with this pattern, how would I override an individual element / node type?

say I wanted to adjust the `h1` - how could I do that?

Reply:
Option 1: just pass an object. This means h1 will be the ONLY converter and no other nodes will be converted. No defaultConverters included
 
```tsx
  <RichText
      converters={{
        h1: // h1Converter
      }}
      editorState={data.lexicalWithBlocks as SerializedEditorState}
    />
```
    
Option 2: spread defaultConverters. As the h1 converter is added AFTER the h1 converter added by defaultConverters, it will take precedence:

```tsx
  <RichText
      converters={({ defaultConverters }) => ({
        ...defaultConverters,
        h1: // h1Converter
      })}
      editorState={data.lexicalWithBlocks as SerializedEditorState}
    />
```

For consistency, this is the exact same pattern used for lexical editor features